src/app/components/availability/availability.directive.js

```diff
@@ -30,6 +30,7 @@ class AvailabilityController {
     this.changePasswordForm.newPasswordConfirm = "";
     this.showChangePassword = false;
     this.datepickerOptions = {
+      "minDate": new Date().setHours(0,0,0),
```

Comment:
Yeah, I know that. I tried to use `"minDate": this.moment().format("YYYY-MM-DD"),`, but it does not work for me...